guides/prereq/standalone/common-configurations/gke.yaml

```diff
@@ -0,0 +1,5 @@
+provider:
+  name: gke
+
+inferencePool:
+  apiVersion: "inference.networking.k8s.io/v1"
```

Comment:
this is the default now, no need to specify

Reply:
updated to have `standalone` in `inference-scheduling` this file no longer exists.